nexus/tests/test_commands.rs

```diff
@@ -165,3 +165,23 @@ fn test_nexus_openapi_internal() {
      */
     assert_contents("../openapi/nexus-internal.json", &stdout_text);
 }
+
+#[test]
+fn test_oxapi_demo_no_args() {
+    let exec = Exec::cmd("../tools/oxapi_demo");
+    let (exit_status, stdout_text, stderr_text) = run_command(exec);
+
+    assert_exit_code(exit_status, EXIT_USAGE);
+    assert!(stdout_text.contains("usage:"));
+    assert_eq!(stderr_text, "oxapi_demo: command not specified\n");
+}
+
+#[test]
+fn test_oxapi_demo() {
+    let exec = Exec::cmd("../tools/oxapi_demo").arg("organizations_list");
+    let (exit_status, stdout_text, stderr_text) = run_command(exec);
+
+    assert_exit_code(exit_status, EXIT_SUCCESS);
```

Comment:
This probably _should_ return a failure code but it works for what we're trying to do!

Reply:
on CI it apparently does!

```
---- test_oxapi_demo stdout ----
thread 'test_oxapi_demo' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`', nexus/tests/test_commands.rs:184:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
I think the behavior here depends on whether Nexus is running.  If the test runs concurrently with another one that starts Nexus, it may succeed by accident.  Maybe we want to either give it a known-bogus URL (which might require changing the script more?) or start a Nexus like other tests.

Reply:
heh

```
---- test_oxapi_demo stdout ----
[nexus/tests/test_commands.rs:182] run_command(exec) = (
    Exited(
        1,
    ),
    "",
    "oxapi_demo: json not found in PATH. Install it with npm i -g json.\n",
)
thread 'test_oxapi_demo' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`', nexus/tests/test_commands.rs:184:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

---- test_oxapi_demo_no_args stdout ----
[nexus/tests/test_commands.rs:172] run_command(exec) = (
    Exited(
        1,
    ),
    "",
    "oxapi_demo: json not found in PATH. Install it with npm i -g json.\n",
)
thread 'test_oxapi_demo_no_args' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `2`', nexus/tests/test_commands.rs:174:5
 ```

Reply:
Ugh.  We should replace this with a Rust tool.

In the meantime, it may not even be worth testing this.

Reply:
You said it, not me!

Reply:
#458